### PR TITLE
Update git-cheatsheet.md with unrealted histories command 

### PR DESCRIPTION
**Pull request by rudrakshkarpe**
### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [X] Search for duplicates.
- [X] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: CheatSheets/git-cheatsheet.md [hunk 1]
```diff
@@ -161,5 +161,9 @@ Table of Contents
 |`git stash pop`| write working from top of stash stack|
 |`git stash drop` | discard the changes from top of stash stack|
 
+## Merge Unrelated histories
+| command | Description |
+| ------- | ----------- |
+| `git merge --allow-unrelated-histories`| Merge unrelated histories|
```
Comment (crescentpartha) [inline CheatSheets/git-cheatsheet.md:167]:
```suggestion
| `git merge --allow-unrelated-histories` | Merge unrelated histories - Allow merging branches with no common history base |
```
Please, add this command in the `Branching & Merging` section.

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Request to modify.

File: CheatSheets/git-cheatsheet.md (hunk 1 above)
Comment (rudrakshkarpe) [reply CheatSheets/git-cheatsheet.md:167]: Alright I'll do that thanks for letting me know

**Comment by rudrakshkarpe** [comment]
> Request to modify.

I've made the required changes please review it. I hope you'll find it useful and I looking forward to merge see my PR merged

**Review by crescentpartha — APPROVED** [review]
Approved.

**Pull request merged**